spec/features/idv/steps/in_person/verify_info_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 require 'axe-rspec'
 
-RSpec.describe 'doc auth IPP VerifyInfo', js: true do
+RSpec.describe 'doc auth IPP VerifyInfo', js: true, allow_browser_log: true do
   include IdvStepHelper
```

Comment:
Again, maybe out of scope so resolve if too details. Address and state id both test `allows the user to cancel and start over` and `allows the user to cancel and return` when visiting.

Reply:
Spec covered- resolving comment